File: programming files/datetime_convert.java
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class datetime_convert {
    public static void main(String[] args) {
        // Original date as a string
        String dateStr = "2022-03-17 10:45:30";
        // Creates a DateTimeFormatter object following the pattern used in the original string
        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
        // Saves LocalDateTime object as dateObj
        LocalDateTime dateObj = LocalDateTime.parse(dateStr, formatter);
        // Takes the dateObj and calls the format method on it, then saves that reformatted date as a string
        String formattedDate = dateObj.format(DateTimeFormatter.ofPattern("MM/dd/yyyy HH:mm:ss"));

        // Prints the formatted date
        System.out.println(formattedDate);
    }
}

File: programming files/datetime_convert.py
from datetime import datetime

# Date to convert
date_str = "2022-03-17 10:45:30"
# Converts the string into a datetime using datetime.strptime and stores it to date_obj
date_obj = datetime.strptime(date_str, '%Y-%m-%d %H:%M:%S')
# Saves the reformatted date to formatted date in the specified format
formatted_date = date_obj.strftime('%m/%d/%Y %H:%M:%S')

# Prints the new formatted date
print(formatted_date)
